Slambook/index.js

```diff
@@ -67,9 +67,32 @@ form.addEventListener('submit',function(e){
     allSlamArray.push(formValue)
     console.log(formValue)
 console.log(allSlamArray)
+showSlamList();
+})
```

Comment:
The variable 'allSlamArray' is not defined in the function. It should be passed as a parameter to the function or defined within the function.